build.sh

```diff
@@ -0,0 +1,22 @@
+ECOV=0483b1e
+CWD=`pwd`
+
+if [ ! -d eco ]; then
+    git clone https://github.com/softdevteam/eco
+    cd eco
+    git checkout ${ECOV}
+    cd ..
+fi
+
+if [ -f results.csv ]; then
+    rm results.csv
+fi
+
+if [ -f performance.pdf ]; then
+    rm performance.pdf
+fi
+
+cp benchmark.py eco/lib/eco/
+cd eco/lib/eco/
+find $CWD/javastdlib5/ -iname \*.java -exec ./benchmark.py {} \;
+mv results.csv $CDW/
```

Comment:
Moving results to the `CWD` seems weird (and `CDW` is a misspelling!). I'd suggest just dumping things in the current directory, since that's what you tell users to do in the README.

Reply:
Again, I'm within the `eco/lib/eco` folder but want to run `plot.py` outside of it. I dunno if there is an easy way to run `eco/lib/eco/benchmark.py` from the top-level directory without the eco dependencies failing.